crates/holochain_types/src/websocket.rs

```diff
@@ -1,14 +1,15 @@
 //! Common types for WebSocket connections.
 
 use itertools::Itertools;
+use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use std::collections::HashSet;
 
 /// Access control for controlling WebSocket connections from browsers.
 /// Anywhere other than a browser can set the `Origin` header to any value, so this is only relevant for browser connections.
 ///
 /// See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) for more information.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, JsonSchema)]
 pub enum AllowedOrigins {
```

Comment:
All these types we reference all over the place really do suggest a problem with API definition... Not a problem for this PR but it seems unhelpful to have mixed-purpose types used like this.